Examples/testCustomEntitlementsComputation/testCustomEntitlementsComputation/ContentView.swift

```diff
@@ -0,0 +1,278 @@
+//
+//  ContentView.swift
+//  testCustomEntitlementsComputation
+//
+//  Created by Andrés Boedo on 4/20/23.
+//
+
+import SwiftUI
+import RevenueCat
+
+struct CustomerInfoStreamFires: Identifiable {
+    let customerInfo: CustomerInfo
+    let id = UUID()
+    let date: Date
+}
+
+struct ContentView: View {
+
+    @State private var streamTask: Task<Void, Never>?
+    @State private var offerings: Offerings?
+    @State private var customerInfoStreamFires: [CustomerInfoStreamFires] = []
+    @State private var showingLogInAlert = false
+    @State private var appUserID = Constants.defaultAppUserID
+    @State private var showingExplanation = false
+
+
+    var body: some View {
+        VStack {
+            Text("Custom Entitlements Computation App")
+                .font(.largeTitle)
+                .padding()
+
+            Button(action: {
+                showingExplanation.toggle()
+            }) {
+                VStack{
+                    Text("This app uses RevenueCat under CustomEntitlementsComputation mode.")
+                        .foregroundColor(.primary)
+                    Text("Tap here for more details about this mode.")
+                        .bold()
+                        .padding()
+                        .foregroundColor(.primary)
+                }
+            }
+
+            HStack {
+                Text("Current App User ID:")
+                    .bold()
+                Text(appUserID)
+            }
+
+            Button(action: {
+                showingLogInAlert = true
+            }) {
+                Text("Login")
+                    .font(.system(size: 20))
+                    .foregroundColor(.white)
+                    .frame(maxWidth: .infinity)
+                    .padding()
+                    .background(Color.blue)
+                    .cornerRadius(20)
+                    .padding()
+
+            }
+
+            Button("Purchase first offering") {
+                Task {
+                    await purchaseFirstOffering()
+                }
+            }
+            .font(.system(size: 20))
+            .foregroundColor(.white)
+            .frame(maxWidth: .infinity)
+            .padding()
+            .background(Color.blue)
+            .cornerRadius(20)
+            .padding()
+
+            .task {
+                Purchases.configureInCustomEntitlementsComputationMode(apiKey: Constants.apiKey,
+                                                                       appUserID: appUserID)
+                do {
+                    self.offerings = try await Purchases.shared.offerings()
+                    print("offerings: \(String(describing: offerings))")
+                }
+                catch {
```

Comment:
Nit per our style guide:
```suggestion
                } catch {
```